docs/DeveloperGuide.md

```diff
@@ -347,13 +347,53 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *` | new user | view more information about a particular command | so that I can learn how to use various commands
+`* * *` | user | set a task by specifying a deadline | know when the task is due
+`* * *` | user | add a task with a start time and end time | record tasks that are events
+`* * *` | user | add a task by specifying a task description only | record tasks that need to be done ��some day��
+`* * *` | user | delete a task | get rid of tasks that I no longer care to track
+`* * *` | user | mark a task as done so that the task will not appear in my to-do list anymore
+`* * *` | user | unmark tasks previously marked as done to edit the task status
+`* * *` | user | edit the deadline of a specific task when the deadline of a task changes
+`* * *` | user | edit the task descriptions | change the descriptions when I get new information
+`* * *` | user | list tasks that are due within the day, week or month | have an overview of my schedule and decide what needs to be done soon
+`* * *` | user | search and list out all the tasks containing the keyword(s)
+`* * *` | user | undo my most recent action so that my mistakes are not permanent
+`* * *` | advanced user | use shorter versions of a command | type a command faster
+`* * *` | advanced user | specify which folder I want to save the files in | have easy access to the tasks just by sharing the files
+`* *` | user | set tasks to repeat over a specified interval | manage recurring tasks
```

Comment:
I think you may have too many level two priority user stories. Try to diversify your ratings (maybe from 1-5) so that you can implement them batch by batch.